### ISSUE TITLE
Youtube previews no longer work.

### ISSUE THREAD

**Issue by p0lm** [keybase/client#24268]
Youtube previews are not loaded, even though link previews are enabled (either explicitely or always) in settings. The problem occurs on both the desktop client as well as the android app. Problem occurs for all members in team.

**Comment by peterrus** [comment]
Seem to be getting fixed: https://github.com/keybase/client/pull/24271

**Comment by p0lm** [comment]
Fixed with #24271

**Comment by 24Gosiaczek1982** [comment]
Super

**Comment by peterrus** [comment]
I thought unfurling was done client-side, but there hasn't been a new client release since this PR was merged, yet the unfurling is working as intended again. Does anyone know why this is?

**Comment by p0lm** [comment]
Seems to be broken again.

**Comment by pastorhudson** [comment]
It seems that the og meta tags are in the <head> again. Did #24271  cause it to not look in the head and only in the body?

**Comment by p0lm** [comment]
> It seems that the og meta tags are in the again. Did #24271 cause it to not look in the head and only in the body?

From what I can see [https://github.com/keybase/client/pull/24271/files#diff-f5a8c5f7c1b29454bfde4a1ff31fc10e](here) it should not. So this hasn't made it into a release yet. 

Guess people nowadays are more busy fixing Zoom's problems. Maybe it's time to move on.

**Comment by ankushnarula** [comment]
It appears YouTube has migrated their metadata to [Open Graph Protocol](https://ogp.me).  

Here's an example snippet from [this YouTube page](https://www.youtube.com/watch?v=WxfA1OSev4c):

```
<meta property="og:site_name" content="YouTube">
<meta property="og:url" content="https://www.youtube.com/watch?v=WxfA1OSev4c">
<meta property="og:title" content="Alex Filippenko: Supernovae, Dark Energy, Aliens &amp; the Expanding Universe | Lex Fridman Podcast #137">
<meta property="og:image" content="https://i.ytimg.com/vi/WxfA1OSev4c/maxresdefault.jpg">
<meta property="og:image:width" content="1280">
<meta property="og:image:height" content="720">
<meta property="og:description" content="Alex Filippenko is an astrophysicist and professor of astronomy at Berkeley. Please support this podcast by checking out our sponsors: - Neuro: https://www.g...">
```
I'm not a `go` developer (let alone familiar with 'gocolly') and also don't have a build environment. But a cursory glance at #24271 suggests the need for an Open Graph tag parser to the `unfurl` package.

**Comment by peterrus** [comment]
Yes, for some reason youtube unfurling started working again recently. But only on linux clients. Not on other platforms.

-------- Original Message --------
